blueprint/api.apib

```diff
@@ -1287,6 +1287,66 @@ A Stripe card belongs to a user.
 
     + Attributes (Record Not Found Response)
 
+# Group Stripe Customers
+
+This resource identifies a customer created on the [Stripe API](https://stripe.com/docs/api#customers).
+
+A Stripe customer can have many subscriptions.
```

Comment:
I would explain a tad more about what we mean by subscriptions, just in the sense that they're a donation to a project.